libs/SalesforceHybridSDK/SalesforceHybridSDK.xcodeproj/project.pbxproj

```diff
@@ -1370,6 +1366,13 @@
 			remoteRef = CEA8844E1C191796008D871B /* PBXContainerItemProxy */;
 			sourceTree = BUILT_PRODUCTS_DIR;
 		};
+		CEEF966C1EB7CBF50059CA79 /* Cordova.framework */ = {
```

Comment:
The static target uses Cordova's static target and the dynamic target uses Cordova's dynamic target.